plugins/discovery-ec2/src/test/java/org/elasticsearch/discovery/ec2/Ec2NetworkTests.java

```diff
@@ -32,6 +32,9 @@
 
 /**
  * Test for EC2 network.host settings.
+ * <p>
+ * Warning: This test doesn't assert that the exceptions are thrown.
```

Comment:
can you open up a issue for this test and assign it to someone? maybe the original author?

Reply:
#35141